src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -277,6 +284,42 @@ protected function getAttributes($object, $format = null, array $context)
         return $attributes;
     }
 
+    /**
+     * @internal this method is wrapper
+     * Gets the attribute value
+     *
+     * @param object      $object
+     * @param string      $attribute
+     * @param string|null $format
+     *
+     * @return mixed
+     */
+    private function getValue($object, $attribute, $format = null, array $context = [])
```

Comment:
You must use typehint here. Then remove useless docblocks.

Reply:
I don't agree with you. If you can, please, give me an example of code.